### ISSUE TITLE
new rule: react-a11y-titles

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#197]
COMPLETED: Title element should not be empty. (http://oaa-accessibility.org/wcag20/rule/13/)

Title content should be concise. (http://oaa-accessibility.org/wcag20/rule/24/)

Title text must contain more than one word. (http://oaa-accessibility.org/wcag20/rule/25/)

References: 
* https://www.w3.org/TR/WCAG20/#navigation-mechanisms
* 2.4.2 Page Titled: Web pages have titles that describe topic or purpose. (Level A)
